hack/proto/Dockerfile

```diff
@@ -0,0 +1,45 @@
+FROM golang:1.16 AS generate-files
```

Comment:
It is recommended to pin dependencies based on `SHA` https://github.com/ossf/scorecard/blob/main/docs/checks.md#pinned-dependencies instead of `tags` because tags can be moved.